lib/class-wp-rest-block-directory-controller.php

```diff
@@ -71,9 +82,11 @@ public function register_routes() {
 	 *
 	 * @since 6.5.0
 	 *
+	 * @param WP_REST_Request $request Full details about the request.
 	 * @return WP_Error|bool True if the request has permission, WP_Error object otherwise.
+	 * phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
 	 */
-	public function permissions_check() {
+	public function get_items_permissions_check( $request ) {
```

Comment:
Looking at the code, it seems like this permissions check is now only used for retrieving a list of plugins, so I wondered whether the `current_user_can( 'activate_plugins' )` check still relevant in this function?